src/mozmlops/templates/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1
```

Comment:
Both good points! I would..._hope_ it would work if someone were, say, testing out this itty bitty baby flow on GPUs 😂 , though I have not tried that so far.

Reply:
Yes! You're correct in that this would work! It would not make use of GPUs though :(